src/ChatGPTModal.tsx

```diff
@@ -38,6 +40,23 @@ export class ChatGPTModal extends Modal {
 		if (this.myRoot) this.myRoot.unmount()
 		contentEl.empty();
 	}
+
+	async saveToAndOpenNewNote(text: string) {
+		const noteRandomId = Math.floor(Math.random() * (100000 - 1) + 1);
```

Comment:
IMO as a followup we should have the title be autogenerated summary of the content...